src/test/run-make/libtest-thread-limit/Makefile

```diff
@@ -0,0 +1,7 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# only-linux
+
+all:
+	$(RUSTC) test.rs --test
+	$(shell ulimit -p 0 && $(call RUN,test))
```

Comment:
My guess is that after setting ulimit to 0 running the binary doesn't work if the shell tries to do so via fork and exec?

Reply:
I added `--target $(TARGET)` and everything is fine (locally) now. :tada: Thanks!

> My guess is that after setting ulimit to 0 running the binary doesn't work if the shell tries to do so via fork and exec?

Everything it's fine when the compilation succeeds. Maybe the it tries to do something when `rustc` fails...

Reply:
If things work locally let's give CI another try.